### PR TITLE
[prometheus-pushgateway] Align resource label handling

### PR DESCRIPTION
**Pull request by skhalash**
Signed-off-by: Stanislav Khalash <stanislav.khalash@sap.com>

#### What this PR does / why we need it:
Most of the templated resources are using the `prometheus-pushgateway.defaultLabels` template to insert labels. The only difference is `Ingress` and `ServiceMonitor`, where the labels are hard-coded. This PR eliminates code duplication by aligning label handling for all resources.

#### Checklist

- [x] [DCO](https://github.com/prometheus-community/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [x] Chart Version bumped
- [x] Title of the PR starts with chart name (e.g. `[prometheus-couchdb-exporter]`)

### CODE REVIEW

**Review by monotek — APPROVED** [review]

**Comment by monotek** [comment]
@scottrigby 
could you jump in with a review / merge please?

**Comment by scottrigby** [comment]
Ah, @cstaud is also a maintainer of this chart but is not invited to the @prometheus-community/helm-charts-maintainers team 😮 I see this was worked around here https://github.com/prometheus-community/helm-charts/pull/195#issuecomment-721769838.

@prometheus-community/helm-charts-admins (@SuperQ or @bismarck) would you please invite him to the helm-charts-maintainers team? Thanks 😄 

Let's wait just a bit to get that sorted or at least give @cstaud a chance to review. I was not aware this never happened, so he was never notified of this PR and hasn't had a chance to review yet.

**Comment by bismarck** [comment]
@scottrigby I don't believe I have permission to add @cstaud to the team?

**Comment by scottrigby** [comment]
@bismarck apologies, autocomplete snafu 🤦‍♂️  I meant @brancz 😄 

I have also opened this issue https://github.com/prometheus-community/community/issues/31

**Comment by skhalash** [comment]
@scottrigby @brancz Hey folks, sorry for bugging you, but did you have a chance to review the PR?

**Comment by SuperQ** [comment]
@scottrigby Sorry, I didn't see the add member issue. Invite sent.

**Review by monotek — DISMISSED** [review]

**Review by zanhsieh — DISMISSED** [review]

**Review by monotek — APPROVED** [review]

File: charts/prometheus-pushgateway/Chart.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: "1.3.0"
 description: A Helm chart for prometheus pushgateway
 name: prometheus-pushgateway
-version: 1.5.2
+version: 1.7.0
```
Comment (zanhsieh) [inline charts/prometheus-pushgateway/Chart.yaml]: 1.7.1

**Review by zanhsieh — CHANGES_REQUESTED** [review]

File: charts/prometheus-pushgateway/Chart.yaml (hunk 1 above)
Comment (skhalash) [reply charts/prometheus-pushgateway/Chart.yaml]: Done

**Review by monotek — APPROVED** [review]

**Review by zanhsieh — APPROVED** [review]

**Comment by zanhsieh** [comment]
@scottrigby 
could you jump in with a review / merge please?

**Comment by scottrigby** [comment]
Hi yes, I believe we've waited long enough per our PROCESSES doc 👌

**Review by scottrigby — APPROVED** [review]

**Comment by scottrigby** [comment]
Can not merge on my phone now - as settings need to be temporarily changed to do so in the GitHub app - can do on laptop tomorrow.

**Comment by skhalash** [comment]
Hey @scottrigby, just a friendly reminder ;) Could you please merge it?

**Comment by scottrigby** [comment]
Done! ✅

**Pull request merged**